examples/image_classifier/image_classifier.py

```diff
@@ -0,0 +1,263 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
**Nit but important enough to request change for:** Metaflow flows canonically end with _flow to distinguish them from model classes and supporting libraries. So in this case it would be image_classifier_flow.py.

Reply:
Done.